sci-mathematics/calc/calc-2.12.6.7.recipe

```diff
@@ -11,18 +11,21 @@ COPYRIGHT="1999-2017 Landon Curt Noll"
 LICENSE="GNU LGPL v2.1"
 REVISION="1"
 SOURCE_URI="http://www.isthe.com/chongo/src/calc/calc-$portVersion.tar.bz2"
-CHECKSUM_SHA256="7e7eba580055e53e57cbe147e1955060b0374234ef9b3ec4f29f1cfd8b06a206"
+CHECKSUM_SHA256="3a4f1acde15941048214f393beb97f9e12fc1ef2585fe0ab026e93ebcd19dd46"
 PATCHES="calc-$portVersion.patchset"
 
 ARCHITECTURES="x86_gcc2 x86 x86_64"
 SECONDARY_ARCHITECTURES="x86_gcc2 x86"
 
+libVersion=2.12.6.7
+libVersionCompat="$libVersion compat >= ${libVersion%%.*}"
+
 PROVIDES="
 	calc$secondaryArchSuffix = $portVersion
 	cmd:calc$secondaryArchSuffix = $portVersion
 	cmd:cscript$secondaryArchSuffix = $portVersion
-	lib:libcalc$secondaryArchSuffix
-	lib:libcustcalc$secondaryArchSuffix
+	lib:libcalc$secondaryArchSuffix = $libVersionCompat
+	lib:libcustcalc$secondaryArchSuffix = $libVersionCompat
```

Comment:
```diff
-	cmd:cscript$secondaryArchSuffix = $portVersion
-	lib:libcalc$secondaryArchSuffix = $libVersionCompat
-	lib:libcustcalc$secondaryArchSuffix = $libVersionCompat
+	lib:libcalc$secondaryArchSuffix = $libVersion
+	lib:libcustcalc$secondaryArchSuffix = $libVersion
```
(It looks like upstream had already removed `cmd:cscript` in `calc-2.12.6.3`.)

Reply:
Yeah, I noticed no cmd:cscript but hp didn't produce an error so I thought I'd left it in :)